src/ansys/dynamicreporting/core/adr_service.py

```diff
@@ -210,14 +210,25 @@ def __init__(
                 self._ansys_installation = ansys_installation
                 # verify new path
                 if not os.path.isdir(ansys_installation):
-                    raise InvalidAnsysPath(ansys_installation)
+                    # Option for local development build
+                    if os.environ.get("CEIDEVROOTDOS") is not None:

```

Comment:
@margalva how about trying to get enve.home()?
This would work in any case where pydynamicreporting is being launched from an Ansys install, and it would work also for ADO runs

Reply:
I understand, but the code you are using right now works only and exclusively in the case of ADO runs, while if pydynamicreporting is being launched from an EnSight install makes sense to give priority to enve.home() (it is what we are doing in PyEnSight).
My suggestion is to cover just more cases in one shot. You can wrap the enve.home() call by a try except clause so that it would work only if enve exists

Reply:
Not quite. If I'm developing on my machine, it's way easier for me to set a couple of env. variables on my machine to point to my local dev build of ADR then to use CEI cpython and update the ansys-dynamicreporting-core that cpython uses every time I make a change to the source code. It's to make things easier when developing both sides on my local machine. 
Unless I'm missing an easy way to do it?

Reply:
I think we're talking about two different situations here.
When running from within the ADO pipeline, yes it'll use the CEI cpython so enve is available and we could go that route or use the env. variables, it's indifferent.

But when I'm developing (either on ADO or github side) and want to run the tests locally to make sure I'm not messing up something that it'll take me 2 days to figure out (as changes need to propagate between the two repos) then it's easier for me to set the env. variables once than to have to use CEI cpython and update the py module there every time there's a change. Unless there's a quicker way to do it in this scenario that I'm missing.

So using the env. variables is killing two birds with one stone for me. I can add the enve way if you think it's better, but I'd still like to retain also the env variable approach as it helps me when working locally.

Reply:
@margalva we can go on with the env variable of course.